workers/unity/Packages/com.improbable.gdk.core/Utility/CommandLineUtility.cs

```diff
@@ -57,6 +87,11 @@ private static bool TryGetConfigValue<T>(Dictionary<string, string> dictionary,
                 var flag = args[i];
                 if (flag.StartsWith("+"))
                 {
+                    if (i + 1 >= args.Count)
+                    {
+                        throw new ArgumentException($"Flag \"{flag}\" requires an argument");
```

Comment:
Can you print all arguments here within the exception? It will help the user debug it